content/en/hosting/costs.md

```diff
@@ -25,9 +25,9 @@ In a production environment, the CHT costs USD$0.10 per active user per month to
 
 Medic hosts a number of production CHT instances in Amazon [Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS). By using [OpenCost](https://www.opencost.io/), Medic can closely monitor real world costs with actual end users. OpenCost was used to separate one deployment's CPU, RAM and disk use from another's despite them all running in a multi-tenant EKS cluster.
 
-By looking at dozens of instances and focusing on the highest usage ones, an accurate representation can be derived of a higher cost a self hosting partner.  
+By looking at dozens of instances and focusing on the highest usage ones, an accurate representation can by a self hosting partner to plan their own hosting.  
```

Comment:
> an accurate representation can by a self hosting partner to plan their own hosting.

I'm not sure what you meant to say here.

Reply:
yikes!  fixed in c06d8c2a